pwndbg/exception.py

```diff
@@ -47,6 +47,14 @@ def handle(name='Error'):
         - ``set exception-verbose on`` enables stack traces.
         - ``set exception-debugger on`` enables the post-mortem debugger.
     """
+
+    # This is for unit tests so they fail on exceptions instead of displaying them.
+    if getattr(sys, '_pwndbg_unittest_run', False) is True:
```

Comment:
This could use the same env var that is used for disabling colors BUT we really want to ensure this code is not skipped if someone forgets to set the envvar...